docs/en/ingest-management/release-notes/release-notes-8.17.asciidoc

```diff
@@ -0,0 +1,218 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.17.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.17.0 relnotes
+
+[[release-notes-8.17.0]]
+== {fleet} and {agent} 8.17.0
+
+Review important information about the {fleet} and {agent} 8.17.0 release.
+
+[discrete]
+[[breaking-changes-8.17.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+{agent}::
+* Install images for {agent} on Debian 10 systems are no longer being made available. Debian 11 is now supported instead. {agent-pull}5847[#5847]
+
+[discrete]
+[[new-features-8.17.0]]
+=== New features
+
+The 8.17.0 release Added the following new and notable features.
+
+{fleet}::
+* Expose advanced file logging configuration in the UI. {kibana-pull}200274[#200274]
+
+{agent}::
+* Add support for pre-existing Active Directory user for unprivileged mode. {agent-pull}5988[#5988] {agent-issue}4585[#4585]
+* Add support for the Filestream integration. This will enable migration from the custom log integration which is based on a log input that is planned for deprecation. https://github.com/elastic/integrations/pull/11332[#11332]
```

Comment:
```suggestion
* Add a new custom [Filestream logs integration](https://github.com/elastic/integrations/tree/main/packages/filestream). This will enable migration from the custom log integration which is based on a log input that is planned for deprecation. https://github.com/elastic/integrations/pull/11332[#11332]. 
```

Reply:
Updated the comment so the link is in Asciidoc format.